views/newsletter/editor.html

```diff
@@ -1416,6 +1416,7 @@ <h1 style="display:none"><%= __('Newsletter Editor') %></h1>
       installedAt: '<%= settings["installed_at"] %>',
       mtaMethod: '<%= settings["mta"]["method"] %>',
     };
+    var mailpoet_woocommerce_settings = <%= json_encode(settings['woocommerce']) %>;
```

Comment:
In `newsletters.html` it's passed like this https://github.com/mailpoet/mailpoet/blob/master/views/newsletters.html#L35. Not important, just saying, we may want to unify it.